.rubocop_todo.yml

```diff
@@ -0,0 +1,32 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2018-10-26 16:44:48 +0200 using RuboCop version 0.60.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Configuration parameters: Max.
+Metrics/AbcSize:
+  Exclude:
+    - 'lib/jekyll-commonmark.rb'
+
+# Offense count: 1
+# Configuration parameters: Max.
+Metrics/CyclomaticComplexity:
+  Exclude:
+    - 'lib/jekyll-commonmark.rb'
+
+# Offense count: 1
+# Configuration parameters: Max, AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, IgnoredPatterns.
+# URISchemes: http, https
+Metrics/LineLength:
+  Exclude:
+    - 'lib/jekyll-commonmark.rb'
```

Comment:
How long is the offending line in this file..?

Reply:
https://github.com/jekyll/jekyll-commonmark/blob/b43c6f13091bbdbfe796ff10e465e82a0a58a41f/lib/jekyll-commonmark.rb#L14

103